app/bundles/CoreBundle/Config/config.php

```diff
@@ -903,6 +903,8 @@
         'tmp_path'                        => '%kernel.project_dir%/var/tmp',
         'theme'                           => 'blank',
         'theme_import_allowed_extensions' => ['json', 'twig', 'css', 'js', 'htm', 'html', 'txt', 'jpg', 'jpeg', 'png', 'gif'],
+        'brand_name'                      => 'Your Company (configurable)',
```

Comment:
Nitpick: When we renamed the key from company name to brand name, shall we also change the default value?
```suggestion
        'brand_name'                      => 'Your Brand (configurable)',
```